target/linux/ath79/dts/qca9558_belkin_f9j1108-v2.dts

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	model = "Belkin AC1750 DB Wi-Fi (F9J1108v2)";
+	compatible = "belkin,f9j1108-v2", "qca,qca9558";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		label-mac-device = &eth1;
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+
+		led_system: system {
+			label = "belkin:blue:system";
+			gpios = <&gpio 20 GPIO_ACTIVE_LOW>;
+		};
+
+		usb {
+			label = "belkin:blue:usb";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&hub_port1>;
+			linux,default-trigger = "usbport";
+		};
+
+		wps-blue {
+			label = "belkin:blue:wps";
+			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+		};
+		
+		wps-amber {
+			label = "belkin:amber:wps";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+	
+	gpio-export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		gpio_usb_power {
+			gpio-export,name = "belkin:usb-power";
```

Comment:
This should be formatted like the led labels, i.e. three parts and two colons, and the model for the first one:
f9j1108-v2:power:usb

Reply:
why? on what reference? how is f9j1108-v2 useful to anyone?

Reply:
That appears to be an old comment. model for LED labels is gone, so the comment does not really apply anymore.

Initially, what I was referring to was probably that the actual model should be used instead of more general "belkin" here. But for now, we need neither.